File: ProgramaTiendaPython/Cliente.py
class Cliente:
    def __init__(self, id_cliente, nombre, direccion):
        self.id = id_cliente
        self.nombre = nombre
        self.direccion = direccion
        self.compras = []

    def get_id(self):
        return self.id

    def get_nombre(self):
        return self.nombre

    def agregar_compra(self, compra):
        self.compras.append(compra)

    def get_compras(self):
        return self.compras

    def getId(self):
        return self.id



    def obtener_informacion(self):
        return f"Id: {self.id} Nombre: {self.nombre} y dirección: {self.direccion}"

File: ProgramaTiendaJava/Cliente.java
import java.util.ArrayList;

public class Cliente {
    private int id;
    private String nombre;
    private String direccion;
    private ArrayList<Compra> compras = new ArrayList<>();

    public Cliente(int id, String nombre, String direccion) {
        this.id = id;
        this.nombre = nombre;
        this.direccion = direccion;
    }

    public int getId() {
        return id;
    }

    public String getNombre() {
        return nombre;
    }

    public void agregarCompra(Compra compra) {
        compras.add(compra);
    }

    public ArrayList<Compra> getCompras() {
        return compras;
    }

    public String obtenerInformacion() {
        return String.format("Id: %d Nombre: %s y direccion: %s", id, nombre, direccion);
    }
}

